Framework/Algorithms/src/MaxEnt.cpp

```diff
@@ -376,7 +376,7 @@ void MaxEnt::exec() {
 * counts
 * @return : Spectrum 'spec' as a complex vector
 */
-std::vector<double> MaxEnt::toComplex(const API::MatrixWorkspace_sptr &inWS,
+std::vector<double> MaxEnt::toComplex(API::MatrixWorkspace_const_sptr &inWS,
```

Comment:
Since you already change things like this, consider going all the way and pass a reference to the workspace.